scrape/manager_test.go

```diff
@@ -829,25 +904,35 @@ func TestManagerCTZeroIngestion(t *testing.T) {
 						got = append(got, f.f)
 					}
 				}
+
 				if len(app.resultFloats) > 0 {
 					return nil
 				}
 				return fmt.Errorf("expected some samples, got none")
 			}), "after 1 minute")
 			scrapeManager.Stop()
 
-			// Check for zero samples, assuming we only injected always one sample.
-			// Did it contain CT to inject? If yes, was CT zero enabled?
-			if tc.counterSample.CreatedTimestamp.IsValid() && tc.enableCTZeroIngestion {
-				require.Len(t, got, 2)
-				require.Equal(t, 0.0, got[0])
-				require.Equal(t, tc.counterSample.GetValue(), got[1])
-				return
-			}
+			switch tc.typ {
+			case "application/vnd.google.protobuf; proto=io.prometheus.client.MetricFamily; encoding=delimited":
+				// Check for zero samples, assuming we only injected always one sample.
+				// Did it contain CT to inject? If yes, was CT zero enabled?
+				if tc.counterSampleProto.CreatedTimestamp.IsValid() && tc.enableCTZeroIngestion {
+					require.Len(t, got, 2)
+					require.Equal(t, 0.0, got[0])
+					require.Equal(t, tc.counterSampleProto.GetValue(), got[1])
+					return
+				}
+
+				// Expect only one, valid sample.
+				require.Len(t, got, 1)
+				require.Equal(t, tc.counterSampleProto.GetValue(), got[0])
 
-			// Expect only one, valid sample.
-			require.Len(t, got, 1)
-			require.Equal(t, tc.counterSample.GetValue(), got[0])
+			case "application/openmetrics-text; version=1.0.0; charset=utf-8":
+				require.Len(t, got, len(tc.exp))
+				for i, e := range tc.exp {
+					require.Equal(t, e.value, got[i])
+				}
+			}
```

Comment:
I think we wanted to check if a metric with _created suffix was ingested if `enabledCTZeroIngestion` was false, right?

I don't see us doing that here

Reply:
yup, just added